content/projects/infrastructure/plugins.adoc

```diff
@@ -0,0 +1,14 @@
+---
+layout: project
+title: "Plugins"
+section: projects
+tags:
+- plugins
+---
+
+== Plugins
+
+=== maven-hudson-dev-plugin
+This Maven plugin is intended to be the single plugin to contain all
+the goals needed for hacking [core](https://github.com/jenkinsci/jenkins).
```

Comment:
This is a markdown link not an asciidoc link.  😭 You'll need to do a bunch more markdown-to-asciidoc processing as part of generating this file.